substrate/frame/revive/src/exec.rs

```diff
@@ -1809,6 +1833,45 @@ where
 		self.run(executable.expect(FRAME_ALWAYS_EXISTS_ON_INSTANTIATE), input_data, BumpNonce::Yes)
 			.map(|_| address)
 	}
+
+	fn instantiate_with_code(
```

Comment:
maybe we can merge this with `instantiate` and make the code arg a Code enum to distinguish between hash / bytecode

Reply:
done